client/jetpack-cloud/sections/partner-portal/credit-card-fields/style.scss

```diff
@@ -0,0 +1,108 @@
+.credit-card-fields {
+	display: none;
+
+	&--is-loaded,
+	&--placeholder {
+		display: block;
+	}
```

Comment:
If you follow the advice I gave you before about hiding this element via JS according to the value of `isStripeFullyLoaded`, you can remove these lines.